espanol/batman/Batman.md

```diff
@@ -1 +1,2 @@
-No puedes ser Batman cuando yo soy el verdadero Batman, sigue y se un [Robin](../robin/robin.md)
\ No newline at end of file
+No puedes ser Batman cuando yo soy el verdadero Batman, sigue y se un [Robin](../robin/robin.md)
+Aunque entiendo que es dificil atreverse; pero recuerda, solo es un salto de [fe](../duda/duda.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.